components/integrations/bots/bot.jsx

```diff
@@ -147,6 +147,25 @@ export default class Bot extends React.PureComponent {
         }
     }
 
+    static matchesFilter(bot, filter, owner) {
+        const username = bot.username || '';
```

Comment:
This stuff is done outside the function. Why not just pass it in?

Reply:
it's done outside of the function only in the case of Bot::render.
however this function is also called from Bots::render to filter out elements, so it needs this logic